dlpy/utils.py

```diff
@@ -2416,6 +2417,38 @@ def create_object_detection_table_no_xml(conn, data_path, coord_type, output, an
         Specifies the size of images to resize.
         If a tuple is passed, the first integer is width and the second value is height.
         Default: (416, 416)
+    check_bbox : bool, optional
+        Specifies whether to check resulting bounding boxes according to image_size.
+        Helps in dropping objects whose bounding boxes would be too small after image resizing.
+        Default : False
+    min_bbox_width : integer, optional
+        Specifies the minimum width for bounding boxes after image resizing.
+        All bounding boxes below this threshold will be dropped.
+        Default : 1
+    min_bbox_height : integer, optional
+        Specifies the minimum height for bounding boxes after image resizing.
+        All bounding boxes below this threshold will be dropped.
+        Default : 1
+    min_bbox_x : integer, optional
+        Specifies the minimum x1 value for bounding boxes after image resizing.
+        All bounding boxes below this threshold will be dropped.
+        Helps in dropping bounding boxes outside of the image.
+        Default : 0
+    min_bbox_y : integer, optional
+        Specifies the minimum y1 value for bounding boxes after image resizing.
+        All bounding boxes below this threshold will be dropped.
+        Helps in dropping bounding boxes outside of the image.
+        Default : 0
+    max_bbox_x : integer, optional
```

Comment:
Should this by maximum x2 and maximum y2?

Reply:
Not sure if I got your question right.
The code will calculate x1,y1,x2,y2 coordinates for the given image size (after rescaling).
max_bbox_x is the threshold for x2.
max_bbox_y is the threshold for y2.

Could you explain your code change in more detail?